File: src/leetcode/easy/twosum1/slow/Solution.py
from typing import List


class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for li in range(0, len(nums) - 1, +1):
            for ri in range(li + 1, len(nums), +1):
                if (nums[li] + nums[ri]) == target:
                    return [li, ri]

File: src/leetcode/easy/twosum1/slow/Solution.js
// Comment me for solution submission
// Uncomment me to run Test.js
// export { twoSum }

/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
function twoSum(nums, target) {
    for (let li = 0; li < nums.length - 1; li++) { // li means left index
        for (let ri = li + 1; ri < nums.length; ri++) {
            let left = nums[li];
            let right = nums[ri];
            if (right + left === target) {
                return [li, ri];
            }
        }
    }
};